readthedocs/vcs_support/backends/hg.py

```diff
@@ -33,7 +33,9 @@ def pull(self):
     def clone(self):
         self.make_clean_working_dir()
         try:
-            output = self.run("hg", "clone", self.repo_url, ".")
+            output = self.run(
```

Comment:
```suggestion
            # https://github.com/readthedocs/readthedocs.org/pull/9042
            output = self.run(
```

Reply:
I added a small comment and link to this PR.